src/params/geo-shape.js

```diff
@@ -15,6 +16,8 @@ class GeoShape extends Json {
     this.clockwise = clockwise;
     this.type = 'array';
     this.items = { type: 'array', items: { type: 'number' } };
+    assert(relaxed === undefined || typeof relaxed === 'boolean');
```

Comment:
not sure about assert placement on all of these

Reply:
Moved them all to be right under constructor.